Bottom: 12e50888352384de5f9ee23f448bf032ab30b794
Top:    f24bbb08acfb4735332ccdb1ec072777fe457b2e
Author: Jon Smirl <jonsmirl@smirl2.(none)>
Date:   2013-06-27 10:11:31 -0400

Refresh of asiarf

---

diff --git a/files/etc/config/wireless b/files/etc/config/wireless
index 4cfe7be..a328f37 100644
--- a/files/etc/config/wireless
+++ b/files/etc/config/wireless
@@ -9,28 +9,46 @@ config wifi-device radio0
 	list ht_capab	SHORT-GI-40
 	list ht_capab	RX-STBC1
 
-config wifi-iface
-        option device 'radio0'
-        option ifname 'ap0'
-        option network 'lan'
-        option mode 'ap'
-        option ssid 'Smirl secure'
-        option encryption 'psk2'
-        option key 'iebgener'
+config wifi-iface 'ap_0'
+	option device 'radio0'
+	option ifname 'ap0'
+	option network 'lan'
+	option mode 'ap'
+	option maxassoc '20'
+	option ssid 'Smirl secure'
+	option encryption 'psk'
+	option key 'iebgener'
+	option hidden '0'
+	option disabled '0'
 
-config wifi-iface
-        option device 'radio0'
-        option ifname 'ap1'
-        option ssid 'Smirl open'
-        option network 'hotspot'
-        option mode 'ap'
-        option encryption 'none'
-        option isolate '1'
+config wifi-iface 'ap_1'
+	option device 'radio0'
+	option ifname 'ap1'
+	option network 'hotspot'
+	option mode 'ap'
+	option maxassoc '20'
+	option ssid 'Smirl open'
+	option encryption 'none'
+	option hidden '0'
+	option disabled '0'
+
+config wifi-iface 'ah_0'
+	option device 'radio0'
+	option ifname 'adhoc'
+	option network 'mesh'
+	option encryption 'none'
+	option bssid '02:CA:FF:EE:BA:BE'
+	option ssid 'vt-mesh'
+	option disabled '0'
+	option mode 'adhoc'
+	option hidden '0'
+
+config wifi-iface 'sta_0'
+	option device 'radio0'
+	option mode 'sta'
+	option network 'wan'
+	option ssid 'host-ssid'
+	option key 'host-passphrase'
+	option encryption 'psk'
+	option disabled '1'
 
-config wifi-iface
-        option device 'radio0'
-        option ifname 'adhoc0'
-        option network 'mesh'
-        option mode 'adhoc'
-        option ssid 'mesh'
-        option bssid '02:CA:FE:CA:CA:40'
